core/dashboard/pages/home.lua

```diff
@@ -1,3 +1,77 @@
+-- Copyright 2020- Teverse.com
+-- Used to display the home screen of the teverse application
+
+local globals = require("tevgit:workshop/library/globals.lua") -- globals; variables or instances that can be shared between files
+
+local function createFlair(parent)
+    local username = parent:child("username").text
+    teverse.http:get("https://teverse.com/api/users/fromUsername/"..username, {
```

Comment:
Unnecessary call to API? Previously the feed endpoint is used in this file, this api call already returns the user's membership.

E.g.
`v.postedBy.beta`
`v.postedBy.membership`

Reply:
Has been corrected.